app/models/isa_tag.rb

```diff
@@ -1,8 +1,8 @@
-class IsaTag < ApplicationRecord
+class ISATag < ApplicationRecord
   validates :title, presence: true
 
-  has_many :template_attributes, inverse_of: :isa_tag
-  has_many :sample_attributes, inverse_of: :isa_tag
+  has_many :template_attributes, inverse_of: :ISATag
+  has_many :sample_attributes, inverse_of: :ISATag
```

Comment:
I think these `inverse_of`s were accidentally changed, they should match the association name in the `TemplateAttribute` etc. models